README.md

```diff
@@ -176,6 +179,37 @@ a source of entropy. If insufficient events have been collected before
 encryption is attempted, an Error is thrown to prevent insecure transmission of
 cardholder data.
 
+## Examples
+
+### Device ID Prefix
+
+RavelinJS prefixes the deviceId with `rjs-` by default. If you wish to use
+something else, for example if you are [upgrading](#upgrading) from a previous
+version and wish to maintain the opaque string format, simply specify your desired
+prefix or omit entirely:
+
+```
```

Comment:
We could add js to the end of the backticks to highlight the syntax. I'd use a GitHub suggestion here but their parser has a bug in it.

Reply:
TIL!